.github/workflows/elixir-build-check.yaml

```diff
@@ -86,3 +86,22 @@ jobs:
           mix-env: ${{ inputs.mix-env }}
       - name: Check for circular dependencies
         run: mix xref graph --label compile-connected --fail-above 0
+  format:
+    name: Validate formatting complies with `mix format`
+    needs: [dependencies]
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - name: Fetch the full repository so we can perform a Git diff
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
`fetch-depth: 0` indicates we should fetch ALL git repo info, which is necessary for the `git diff` command below.

Reply:
The checkout by `actions-services-elixir-deps` is controlled by its `skip-checkout` parameter. Should this parameter be used below, since the checkout is done here first?

Reply:
Good call, yes probably. The only other solution would be to update `actions-services-elixir-deps` to take a `fetch-depth` option. I'm not sure it is worth the hassle.

Reply:
PR updated.

Reply:
With this change applied the build fails. Investigating why. https://github.com/rentpath/callrail-service/pull/38

Reply:
PR updated. It turns out `--no-run-if-empty` xarg flag is required when not on Mac OS.